playbook.md

```diff
@@ -1226,23 +1084,19 @@ But there are some limitations. Under the support service, we do not:
   if we've spent a couple of hours on an updated-related bug and it's still not fixed,
   we're probably dealing with one.
 * From time to time, we may make an exception to these restrictions. If you
-  think that might be appropriate, ask Harry or a delivery lead. In all the
+  think that might be appropriate, ask a delivery lead. In all the
   above cases, we can offer to quote for the required work.
 
-Finally, there is an overarching principle that we don't do anything on a ticket
-that is very time consuming or that is objectively a bad idea. If you think
-you're in this situation, ask a delivery lead for advice.
-
 #### Charging for ticket work
 
 If you decide that a ticket asks for work to be done which falls outside this
 scope, then the work is chargeable. In this situation, we reassign the ticket to
-a delivery lead with an explanation.
+a delivery lead or the business development manager with an explanation.
```

Comment:
Might be better to say 'someone from the client services team' rather than biz dev manager to make sure that this isn't just one person.